docs/usage/search.md

```diff
@@ -0,0 +1,38 @@
+# Search
+
+The search interface uses the ElasticSearch index to search through all publications.
+
+[Elastic Search Query Syntax](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-syntax) applies.
+
+The following fields are indexed:
+
+| Field             | Type            |
+|-------------------|-----------------|
+| id                | Integer         |
+| thinktank_id      | Integer         |
+| title             | String          |
+| subtitle          | String          |
+| ris_type          | String          |
+| authors           | List[String]    |
+| abstract          | String          |
+| publication_date  | String          |
+| last_access       | String/Datetime |
+| url               | String          |
+| pdf_url           | String          |
+| pdf_pages         | Integer         |
+| doi               | String          |
+| isbn              | String          |
+| tags              | List[String]    |
+| created           | String/Datetime |
+| thinktank.name    | String          |
+
+Any word without a field qualifier is searched in all fields.
```

Comment:
not quite true, you have to tell elastic explicitly which fields to use for unqualified searches and we use:
```
TRANSLATION_FIELDS = [
    'title',
    'subtitle',
    'abstract',
]

  def get_search_fields(language):
      return [*get_translation_fields(language, TRANSLATION_FIELDS), 'authors', 'isbn', 'tags']
```

Reply:
should we add `categories` to the defaults?

Reply:
no since categories are a separate feature. more likely we should remove tags because both could be added to the search by clicking on a list.